services/portfolios_ca/src/routes/targets.ts

```diff
@@ -0,0 +1,86 @@
+import { Router } from 'restify-router';
+import { EHFunction } from '../types/config';
+import { TargetsServiceI } from '../types/targets';
+
+export default (errorHandler: EHFunction, TargetsService: TargetsServiceI) => {
+  const router = new Router();
+
+  /**
+   * @apiGroup s_portfolios_ca
+   * @api {get} /portfolios-ca/targets/values PortfoliosByTarget
+   * @apiName PortfoliosByTarget
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Values of all portfolios for a specific target within a given area
+   *
+   * @apiParam (Query params) {String} areaType area type
+   * @apiParam (Query params) {String|Number} areaId area id
+   * @apiParam (Query params) {Number} targetId target id to filter results
+   *
+   * @apiSuccess {Object} result
+   * @apiSuccess {Number} result.target_id target_id target id
+   * @apiSuccess {String} result.target_name target_name target name
+   * @apiSuccess {Number} result.target_national target_national target national value
+   * @apiSuccess {String} result.target_units_short target_units_short units of target national value (short name)
+   * @apiSuccess {String} result.target_units target_units units of target national value (long name)
+   * @apiSuccess {Object[]}  result.portfolios_data portfolios_data Values of portfolios in the selected target
+   * @apiSuccess {Number} result.portfolios_data.id id portfolio id
+   * @apiSuccess {String} result.portfolios_data.name id portfolio name
+   * @apiSuccess {String} result.portfolios_data.short_name id portfolio short name
+   * @apiSuccess {Number} result.portfolios_data.value id portfolio value
+   *
+   * @apiExample {curl} Example usage:
+   *  /portfolios-ca/targets/values?areaType=ea&areaId=CARDER&targetId=1
+   *
+   * @apiUse PortfoliosByTargetExample
+   */
+  router.get(
+    '/portfolios-ca/targets/values',
+    errorHandler((req, res, next) => {
+      if (!(req.params.areaType && req.params.areaId && req.params.targetId)) {
+        const error = { code: 400, message: 'areaType, areaId and targetId are required' };
+        throw error;
+      }
+      return TargetsService.getPortfoliosByTarget().then((value) => {
+        res.send(value);
+        next();
+      });
+    }),
+  );
+
+  /**
+   * @apiGroup s_portfolios_ca
+   * @api {get} /portfolios-ca/targets/list TargetsList
```

Comment:
Add this endpoint to the excel

Reply:
Also, maybe the /list part is a little redundant, but I'm ok with it

Reply:
Done!